package.json

```diff
@@ -30,6 +30,10 @@
     "react": "*",
     "react-native": "*"
   },
+  "optionalDependencies": {
+    "ajv": "^6.5.3",
+    "react-dom": "^16.5.0"
+  },
```

Comment:
why do we need optionalDependencies? react-dom is not really necessary for RN

Reply:
just for making npm/yarn happy, no warning.

Reply:
should I remove that?

Reply:
yes